README.md

```diff
@@ -2,6 +2,8 @@
 
 [![MIT License](https://img.shields.io/badge/License-MIT-brightgreen.svg)](LICENSE)
 
```

Comment:
I recommend removing this blank line so the badges are rendered on the same line.

```suggestion
```